src/extension.ts

```diff
@@ -219,11 +219,11 @@ function runInRlsMode(
 ): void {
     const methodLogger = logger.createChildLogger('runInRlsMode: ');
 
-    const env = configuration.getRlsEnv();
+    const env = configuration.getRlsEnv() || {};
 
     methodLogger.debug(`env=${JSON.stringify(env)}`);
 
-    const args = configuration.getRlsArgs();
+    const args = configuration.getRlsArgs() || [];
```

Comment:
```
child_process.spawn(command[, args][, options])
                              ^
                              if args is undefined, options are ignored.
```

This is how JS optional parameters work.

By the way, you can try with `args: null`. This will crash for your current codebase, as an indicator of lacking defaults.